### PR TITLE
fix first frame reporting logic

### PR DESCRIPTION
**Pull request by yjbanov**
/cc @Hixie

### CODE REVIEW

File: packages/flutter/lib/src/widgets/binding.dart [hunk 1]
```diff
@@ -206,7 +206,7 @@ abstract class WidgetsBinding extends BindingBase implements GestureBinding, Ren
     buildOwner.finalizeTree();
     // TODO(ianh): Following code should not be included in release mode, only profile and debug modes.
     if (_needToReportFirstFrame) {
-      if (_thisFrameWasUseful) {
+      if (!_thisFrameWasUseful) {
         _thisFrameWasUseful = true;
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/binding.dart]: rather than negating the condition, switch the two branches around.

Also, oops. :-|

**Comment by Hixie** [comment]
LGTM

**Comment by yjbanov** [comment]
Stupid GitHub. It showed me your LGTM message, but not the comment. I'll follow up.

**Comment by yjbanov** [comment]
[There](https://github.com/flutter/flutter/pull/3641/files)

**Pull request merged**